### PR TITLE
Lw/bugfixes

### PR DESCRIPTION
**Pull request by phi-lira**
- Fixed material upgrader softlocks editor
- Fixed issue that material upgrader causing material upgrader to proceed even by clicking on dialog x button
- Fixed issue that renderscale was not being applied to screenspace shadow occlusion texture
- Added _ScaledScreenParams shader property to access camera scaled size
- Excluded d3d11_9x from shader targets as we don't support it anymore
- Fixed warning in Particles PBS shader
- Added depth pass to Standard Unlit shader

### CODE REVIEW

**Comment by phi-lira** [comment]
Will reopen this targeting the LWRP branch instead.

File: ScriptableRenderPipeline/Core/CoreRP/Editor/MaterialUpgrader.cs [hunk 1]
```diff
@@ -172,7 +182,7 @@ static void SaveAssetsAndFreeMemory()
 
         public static void UpgradeProjectFolder(List<MaterialUpgrader> upgraders, string progressBarName, UpgradeFlags flags = UpgradeFlags.None)
         {
-            if (!EditorUtility.DisplayDialog("Material Upgrader", "The upgrade will overwrite materials in your project. " + projectBackMessage, "Proceed", "Cancel"))
+            if (!EditorUtility.DisplayDialog(DialogText.title, "The upgrade will overwrite materials in your project. " + DialogText.projectBackMessage, DialogText.proceed, DialogText.cancel))
```
Comment (aleks01010101) [inline ScriptableRenderPipeline/Core/CoreRP/Editor/MaterialUpgrader.cs]: Nitpicking: Dialog contents string could be moved to DialogText as well.

File: ScriptableRenderPipeline/LightweightPipeline/LWRP/LightweightPipeline.cs [hunk 2]
```diff
@@ -657,7 +650,7 @@ private void SetupFrameRenderingConfiguration(out FrameRenderingConfiguration co
 
             bool hdrEnabled = m_Asset.SupportsHDR && m_CurrCamera.allowHDR;
             bool intermediateTexture = m_CurrCamera.targetTexture != null || m_CurrCamera.cameraType == CameraType.SceneView ||
-                m_Asset.RenderScale < 1.0f || hdrEnabled;
+                GetRenderScale() < 1.0f || hdrEnabled;
```
Comment (aleks01010101) [inline ScriptableRenderPipeline/LightweightPipeline/LWRP/LightweightPipeline.cs]: Do we need an intermediate texture if RenderScale is set to >1.0? I suppose we don't create a render target that's larger than the screen...

File: ScriptableRenderPipeline/LightweightPipeline/LWRP/Shaders/LightweightStandardUnlit.shader [hunk 3]
```diff
@@ -31,6 +31,9 @@ Shader "LightweightPipeline/Standard Unlit"
             HLSLPROGRAM
             // Required to compile gles 2.0 with standard srp library
             #pragma prefer_hlslcc gles
+            #pragma exclude_renderers d3d11_9x
+
+            #pragma exclude_renderers d3d11_9x
```
Comment (aleks01010101) [inline ScriptableRenderPipeline/LightweightPipeline/LWRP/Shaders/LightweightStandardUnlit.shader]: Exclude twice

**Pull request merged**